README.md

```diff
@@ -1,32 +1,143 @@
-# Class
+
+# Java Class :  Encapsulating Object State & Behavior 
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Define common characteristics of objects including state and behavior
+- Define fundamental Java class members: fields, constructors, and methods
+
+## Classes in Java
+
+Java is an object-oriented programming language. This means it thinks of the
+world as "objects". Java can model a physical object such as a dog, bike, or television.
+Java can also model abstract objects such as an event (birthday party, 
+music concert, company meeting), the components of a web browser (menu bar, button, 
+popup dialog, scrollbar), or even a programming language (Java, Python, Javascript).
+
+All objects have **state** and **behavior**.   
+- State is the relevant properties (data) about an object. 
```

Comment:
It could just be me... but something about this sentence isn't reading properly. I think it's because you have "is" and then a plural ("properties") right after it?

Reply:
Would this be better:  "State encapsulates the relevant properties ...."